paimon-core/src/test/java/org/apache/paimon/catalog/CatalogTestBase.java

```diff
@@ -948,4 +948,16 @@ public void testFormatTable() throws Exception {
                 .isInstanceOf(Catalog.TableNotExistException.class);
         assertThat(catalog.getTable(newIdentifier)).isInstanceOf(FormatTable.class);
     }
+
+    @Test
+    public void testTableUUID() throws Exception {
+        catalog.createDatabase("test_db", false);
+        Identifier identifier = Identifier.create("test_db", "test_table");
+        catalog.createTable(identifier, DEFAULT_TABLE_SCHEMA, false);
+        Table table = catalog.getTable(identifier);
+        String uuid = table.uuid();
+        assertThat(uuid).startsWith(identifier.getFullName() + ".");
```

Comment:
This will fail those non-FileStoreTable, right?

Reply:
Yes